src/general/OptionsMenu.cs

```diff
@@ -162,6 +162,11 @@ public class OptionsMenu : Control
     [Export]
     public NodePath CustomUsernamePath;
 
+    // Sound tab statics
+    private static List<string> languages = TranslationServer.GetLoadedLocales().Cast<string>()
```

Comment:
This triggers too soon (before the first actual usage). Still please look at how Constants.cs does it:
https://github.com/Revolutionary-Games/Thrive/blob/cf56b89a03d8dbf5067a260d14e319170fc6d1b4/simulation_parameters/Constants.cs#L544
https://github.com/Revolutionary-Games/Thrive/blob/cf56b89a03d8dbf5067a260d14e319170fc6d1b4/simulation_parameters/Constants.cs#L539

Doing it like that *only* causes the static variable to initialize when the property is *first* read, and not during static variable initialization phase.

Reply:
I'd write something like this to make sure the languages only get fetched when required:
```cs
private List<string> languages;
private List<string> Languages => languages ??= FetchLanguages();
private List<string> FetchLanguages() {
    return TranslationServer.GetLoadedLocales().Cast<string>()
        .OrderBy(i => i, StringComparer.InvariantCulture)
        .ToList();
}
```

Reply:
That's not thread safe. See my example in Constants.cs
I looked this into a lot when I wrote that, and that is the *only* thread safe single initialization on demand for a variable without using Lazy I could find.